src/core/xref.js

```diff
@@ -38,6 +37,8 @@ const profiles = {
 
 export const API_URL = "https://respec.org/xref/";
 
+export const informativeRefsInNormative = [];
```

Comment:
Not sure how TypeScript types behave right now, but should add this to avoid `any[]`. (something like this, I have not checked below in IDE).
```suggestion
/** @type {{ term: string; spec: string; element: HTMLAnchorElement }[]} */
export const informativeRefsInNormative = [];
```